### DISCUSSION TITLE
i'm confused

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#83] [Sneak peeks]
why do we have two categories for the same thing...

"sneak peeks" and "testing"

**Comment by rubiidev18alt** [comment]
Sneak peaks are for the sneak peaks for the community, testing is just for admins/contributors to share their sneak peaks

**Reply by raynepaws** [reply]
ok, but anybody can view both, so what's the point?

**Reply by rubiidev18alt** [reply]
I don't know how to make a category available to a group of ppl only

**Reply by raynepaws** [reply]
you can't
